packages/utils/src/utils/defaultSnapshotConfigs.ts

```diff
@@ -4,6 +4,7 @@ import stateConfigJson from './defaultSnapshots/stateConfig.json';
 import type { SnapshotConfigs } from './types';
 
 export const defaultSnapshotConfigs: SnapshotConfigs = {
+  // @ts-expect-error Getting type errors as we can't gen the fuel schema for v4 gas costs
```

Comment:
Need to gen the `v4` gas costs schema

Reply:
@danielbate Was this fixed at the `fuel-core` graphql schema on `0.32.0`? 

It is still unclear to me why they did not add new values for the `GasCostsVersion` enum

Reply:
It was not fixed however I've managed to manually fix the types so we no longer need the `@ts-expect-error`. I'll ask about the enum.